cmdline.h

```diff
@@ -1,3 +1,3 @@
 
 
-int cmdline_main(int argc, const char * argv[]);
+int cmdline_main(int argc, char * const *argv);
```

Comment:
What's the purpose of this change?

Reply:
This matches the type required by getopt_long(), so putting it here avoids having to use a typecast.